doc/setup_instructions.md

```diff
@@ -133,121 +138,25 @@ Attach. Whenever you want to debug, make sure _Wait for Debugger_ is
-package manager to install it.
-
-On windows it is recommended to use RubyInstaller, version 2.4 or
-newer, when installing make sure to also install the MSYS option in
-order to be able to install gems.
-
-After installing ruby open a terminal / command prompt and run:
-
-```sh
-gem install os colorize rubyzip json sha3
-```
-
-On Linux you might need to run the command with `sudo`.
-
-
-If you have trouble installing sha3 on windows: make sure you have
-ruby 2.4 or newer installed with ruby installer for windows. Then run
-`ridk install` and try all of the options. The third option at least
-should reinstall all the ruby development tools, including gmp, which
-is needed for sha3. After that your ruby native extension build tools
-should be installed and the gem installation should work.
-
-If it still doesn't work run `ridk exec pacman -S gmp-devel` and then
-run `ridk install` again.
-
-
-### 7zip
-
-On Linux and mac, use your package manager to install, it is probably
-named `p7zip`.
-
-On Windows download the [official](http://www.7-zip.org/download.html)
-installer release. After installing, add the installed folder (where
-`7z.exe` is) to PATH environment variable. Confirm by running `7z.exe`
-in command prompt or powershell, it should print 7zip version info and
-command line usage.
-
-
-Linters
--------
-
-Thrive uses automatic formatting of source code to keep style as
-consistent as possible.
-
-Note: the following tools is not currently used
-
-For this reason you need to install some
-additional tools.
-
-[NodeJS](https://nodejs.org/en/download/). If you are on Linux you
-should use your OS's package manager to install nodejs.
-
-After installing nodejs install the linter packages with this command:
-```sh
-npm install -g jsonlint
-```
-
-When you are getting ready to commit you should run `ruby
-check_formatting.rb` in order to automatically run all of the
-formatting tools. Make sure that that script doesn't report any errors
-before committing.
-
-
 Building Thrive
 ===============
 
-Make sure you installed the non-optional prerequisites first!
-
 Clone
 -----
 
-Open a terminal on Linux or a command prompt (or powershell) on
-Windows to a folder where you want to place the Thrive folder.
-
-Note: a path with spaces in it MAY NOT WORK, so to avoid issues you
-should choose a folder like `~/projects` or `C:/projects`. Also, long
-paths may cause issues on Windows. One additional potential problem is
-non-English characters in the path name, for example if your path
-includes "työpöytä", the setup will fail.
-
-Windows tip: shift right-click in a folder and select "Open command
-prompt here" or "Open powershell here" to open a cmd window to the
-folder.
-
-And now run:
-
-```
-git clone https://github.com/Revolutionary-Games/Thrive.git
-cd Thrive
-```
+On Github fork Thrive so you have your own repository to work with,
+After that clone your repository onto your computer.
```

Comment:
I'm going to guess that there should be much more indepth instructions on making a clone (you need to be logged in to Github first), also if someone just wants to play the latest version, this is unnecessary.